generator/cli.py

```diff
@@ -28,9 +29,9 @@
 def make_cli_progress_callback():
     """Return a callback that displays progress updates to the console."""
 
-    def _callback(percent: float, message: str = None):
+    def _callback(percent: float, message: str):
```

Comment:
This message is required by `_report_progress` in generator/worker/progress.py which is the only place the callback is used, so making message a required attribute here.